### PR TITLE
remove the "Project" tab from the sage 300 mapping page

### PR DESCRIPTION
**Pull request by JustARatherRidiculouslyLongUsername**
- remove the underlying api call to `/sage-desktop-api/api/workspaces/:id/mappings/settings/`

## Summary by CodeRabbit

- **Refactor**
  - Simplified the page setup logic for the Sage300 mapping component, enhancing performance and reducing loading time.
  
- **Bug Fixes**
  - Improved item filtering by including the 'PROJECT' condition, ensuring more accurate data display.

### CODE REVIEW

File: src/app/integrations/sage300/sage300-main/sage300-mapping/sage300-mapping.component.ts [hunk 1]
```diff
@@ -35,20 +35,8 @@ export class Sage300MappingComponent implements OnInit {
 
   private setupPage(): void {
     this.isLoading = true;
-    this.mappingService.getMappingSettings().subscribe((response) => {
```
Comment (ashwin1111) [inline src/app/integrations/sage300/sage300-main/sage300-mapping/sage300-mapping.component.ts:38]: I'll tell a case that might break this - 

Enable ccc export on staging
You should see a new mapping page named "Corporate Card"

Do the same locally with the current fix - see if the "Corporate Card" mapping page is showing up. If not, we should fix the solution

Comment (JustARatherRidiculouslyLongUsername) [reply src/app/integrations/sage300/sage300-main/sage300-mapping/sage300-mapping.component.ts:38]: You're right - "Corporate Card" won't show up. Fixing now.

**Comment by JustARatherRidiculouslyLongUsername** [comment]
Should be working now @ashwin1111

File: src/app/integrations/sage300/sage300-main/sage300-mapping/sage300-mapping.component.ts [hunk 2]
```diff
@@ -37,8 +37,9 @@ export class Sage300MappingComponent implements OnInit {
     this.isLoading = true;
     this.mappingService.getMappingSettings().subscribe((response) => {
       if (response.results && Array.isArray(response.results)) {
+        console.table(response.results);
```
Comment (ashwin1111) [inline src/app/integrations/sage300/sage300-main/sage300-mapping/sage300-mapping.component.ts:40]: remove loggers

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**